Makefile

```diff
@@ -0,0 +1,31 @@
+#APP = $(foreach file, $(wildcard *.json), $(subst .json,.app,$(file)))
```

Comment:
`Makefile` seems unnecessary, apps are build automatically with buildbot. If you plan to distribute a bundle nonetheless, it might be a good idea to embed info about runtime and app repositories as described in `flatpak-build-bundle` man page.